_posts/addons/2017-10-23-tcp-addon.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Scalingo TCP Addon
+modified_at: 2017-10-23 00:00:00
+category: addons
+tags: tcp load balancer addon custom
+---
+
+## Introduction
+
+By default, Scalingo will only let you run web application. Sometimes this is
+not enough and you will need to have application that can accept raw TCP
+connections.
+
+This addon will let you deploy application that can accept raw TCP connections.
+
+## Setup of the addon
+
+### Provision the addon
+
+First, you need to add the TCP addon to your application. This can be done
+through the dashboard on the addons section of your app under the `Network`
+category.
+
+### Setup configuration
+
+In order to use the Scalingo TCP addon, you will need to define a new container
+type. This can be done in your [`Procfile`](/internals/procfile.html)
+
+
+Your procfile should look like:
```

Comment:
Here is a sample Procfile for a NodeJS application:

Reply:
Done